### PR TITLE
Cleanup copy/pasta and use new codemirror-blocks toolkit API

### PR DESCRIPTION
**Pull request by pcardune**
The only thing more satisfying than writing a bunch of code is deleting a bunch of code. This PR deletes a bunch of files that were never being used, and a bunch of other files that no longer need to be used with the new toolkit API.

TODO before this can be merged:

- [ ] Wait for the toolkit branch in codemirror-blocks to be merged: https://github.com/bootstrapworld/codemirror-blocks/pull/374
- [x] Figure out what the story is with css bundling. Currently it's bundled with the minified js bundle but sounds like it shouldn't be?
- [x] Figure out what the story is with the "*-test-old.js" files in spec/ui as they are currently not being run.

### CODE REVIEW

**Comment by pcardune** [comment]
@schanzer couple questions for you:
1. what's the story with the "spec/ui/*-test-old.js" files? Do they need to run? Do they need to pass?
2. do we want to generate a blocks.css file during the build process? The code to do it has been commented out but it looks like wescheme.org is still loading one?

File: webpack.config.js [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+    commonjs: 'plt.compiler',
+    commonjs2: 'plt.compiler',
+    root: ['plt','compiler'],
+  },
+  'types': 'types',
+  'structs': {
+    commonjs: 'plt.compiler',
+    commonjs2: 'plt.compiler',
+    root: ['plt','compiler'],
+  },
```
Comment (pcardune) [inline webpack.config.js:50]: This configuration is a great example of why bundling should happen as part of the build process for wescheme.org, and not in this repo. This configuration is super-duper specific to the way wescheme.org works, and the resulting bundle would be unusable in any other context.

Comment (schanzer) [reply webpack.config.js:50]: 1. The spec/ui/*-test-old.js files can be safely removed from this repo. They _should_, however, be resurrected in the CMB core repo. When we switched Drag-n-Drop libraries a while back, a lot of these tests broke.
2. See my comment in CMB re: generating a separate CSS file
3. You're right that this configuration is crazy-specific to wescheme. But then again, (a) the WeScheme repo itself has a build system designed in 2008 and isn't getting that love anytime soon and (b) this language repo is specific to WeScheme as a use-case anyway. Do you think this is sufficiently offensive that it's worth overhauling the build process for another repo altogether?

Comment (pcardune) [reply webpack.config.js:50]: 1. ok, I created a separate issue for test files.
2. ok, I'm just going to leave the css files bundles with js.
3. I don't think you have to overhaul anything. I think you could just delete this repo altogether and make it a subdirectory of the wescheme repo. That would make it more clear that this was application code and not some kind of reusable library.

Comment (schanzer) [reply webpack.config.js:50]: That is an _excellent_ idea. I'll add it as an issue.

**Pull request merged**